src/main/java/org/broadinstitute/hellbender/tools/exome/SegmentUtils.java

```diff
@@ -302,7 +351,32 @@ public static void writeACNVModeledSegmentFile(final File outFile,
                                 .append(String.format("%6.4f", segment.getMinorAlleleFractionPosteriorSummary().lower())
                                         .replaceAll("\\s+", ""))
                                 .append(String.format("%6.4f", segment.getMinorAlleleFractionPosteriorSummary().upper())
-                                        .replaceAll("\\s+","")));
+                                        .replaceAll("\\s+", "")));
+    }
+
+    /**
+     * Read ACNV file into ACNV modeled segments.
+     *
+     * @param acnvSegFile File in the ACNV format (*-final.seg).  Must be readable file and never {@code null}
```

Comment:
Actually, all sim-*.seg files can be read (but I guess we might decide to suppress output of intermediate files in the near future).